modules/analysis/cohort.py

```
@@ -119,50 +135,552 @@ def get_user_sub_info_by_year(info_db_no, user_info, user_sub_info, year):
+        try:
+            save_pcl_csv_to_s3(info_db_no, p_retention, c_retention, l_retention)
+            return jsonify({
+                "success": True,
+                "message": "CSV 파일이 S3에 저장되었습니다.",
+            }), 200
+        except Exception as e:
+            print(f"CSV 저장 중 오류 발생: {e}")
+            return jsonify({"success": False, "error": str(e)}), 500
+
+
+def analysis_cohort_SubscriptionType(info_db_no, target_table, year, target_month):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)